Dalamud/Dalamud.cs

```diff
@@ -135,6 +136,9 @@ public void DisposePlugins()
         // will not receive any windows messages
         Service<DalamudIME>.GetNullable()?.Dispose();
 
+        // this must be done before unloading interface manager, since it relies on the window handle members.
+        Service<DragDropManager>.GetNullable()?.Dispose();
```

Comment:
Is there a reason you're not depending on InterfaceManager in DragDropManager instead? That should satisfy the condition in your comment. You can also use the InherentDependencyAttribute.